### ISSUE TITLE
update command line tools

### ISSUE THREAD

**Issue by wkuckro** [mas-cli/mas#58]
The current functionality doesn't allow for an update of Xcode's Command Line Tools. There isn't an update feature from the installer `xcode-select` so you have to use the App Store to do so. 

Command Line Tools also don't show up in `man list` so there isn't a way to download them manually as far as I can tell.

**Comment by rodionovd** [comment]
Thanks for getting in touch @wkuckro!

**TL;DR:** `mas` is only able to install/update *applications* that are listed in the Mac App Store itself. Use `softwareupdate(8)` utility for downloading system updates (including Xcode CLT).

--------

The point of confusion is that Mac App Store provides updates not only for installed applications but also for system components such as Xcode Command Line Tools, iTunes, etc. These actually have nothing to do with MAS itself (it's just used as a nice user-friendly interface), so `mas` can't deal with them.

Luckily for us, there's a built-in utility `softwareupdate` which is able to list new system software updates and install them via command line interface 👌🏼

Here's an example output of `softwareupdate -l`:

```
$ softwareupdate -l
Software Update Tool
Copyright 2002-2015 Apple Inc.

Finding available software
Software Update found the following new or updated software:
   * Command Line Tools (macOS Sierra version 10.12) for Xcode-8.1
	Command Line Tools (macOS Sierra version 10.12) for Xcode (8.1), 123638K [recommended]
   * iTunesXPatch-12.5.2
	iTunes (12.5.2), 145964K [recommended]
```

The man page for this tool contains even more information and usage examples.
